source/includes/examples/tf-example-third-party-integration-stagingprod.rst

```diff
@@ -0,0 +1,28 @@
+.. code-block:: shell 
+   :copyable: true 
+
+   resource "mongodbatlas_third_party_integration" "test_third_party_integration" {
+     project_id = var.atlas_project_id
+     type = "DATADOG"
+     api_key = var.datadog_api_key
+     region = var.datadog_region
+   }
+
+   resource "mongodbatlas_third_party_integration" "test_prometheus" {
+     project_id        = var.atlas_project_id
```

Comment:
remove `atlas_` ?

Reply:
This is how this is set in the variables file.